frontend/src/bundles/friends/pages/friends/friends.tsx

```diff
@@ -0,0 +1,143 @@
+import { Loader } from '~/bundles/common/components/components.js';
+import { DataStatus } from '~/bundles/common/enums/enums.js';
+import {
+    useAppDispatch,
+    useAppSelector,
+    useCallback,
+    useEffect,
+    useState,
+} from '~/bundles/common/hooks/hooks.js';
+import {
+    FriendDetails,
+    TabContent,
+    Tabs,
+} from '~/bundles/friends/components/components.js';
+import { TabsFollowers } from '~/bundles/friends/enums/enums.js';
+import { actions as friendsActions } from '~/bundles/friends/store/friends.js';
+import { type UserFollowingsResponseDto } from '~/bundles/friends/types/types.js';
+
+const Friends: React.FC = () => {
+    const dispatch = useAppDispatch();
+
+    const tabs = [TabsFollowers.FIND_FOLLOWINGS, TabsFollowers.MY_FOLLOWINGS];
+    const [selectedCard, setSelectedCard] =
+        useState<UserFollowingsResponseDto | null>(null);
+
+    const [activeTab, setActiveTab] = useState<string>(
+        TabsFollowers.FIND_FOLLOWINGS,
+    );
+
+    const users = useAppSelector((state) => state.friends.users);
+    const loading = useAppSelector((state) => state.friends.dataStatus);
+
+    const handleSelectCard = useCallback(
+        (user: UserFollowingsResponseDto | null): void => {
+            setSelectedCard(user);
+        },
+        [setSelectedCard],
+    );
+
+    const handleTabClick = useCallback(
+        (tab: string): void => {
+            setActiveTab(tab);
+            setSelectedCard(null);
+        },
+        [setActiveTab, setSelectedCard],
+    );
+
+    const handleAddFollowing = useCallback(
+        (id: number): void => {
+            const addFollowing = async (id: number): Promise<void> => {
+                await dispatch(
+                    friendsActions.addFollowing({ followingId: id }),
+                );
+            };
+            void addFollowing(id);
+        },
+        [dispatch],
+    );
+
+    const handleRemoveFollowing = useCallback(
+        (id: number): void => {
+            const removeFollowing = async (id: number): Promise<void> => {
+                await dispatch(
+                    friendsActions.removeFollowing({ followingId: id }),
+                );
+            };
+            void removeFollowing(id);
+        },
+        [dispatch],
+    );
```

Comment:
I don't think that we need that additional arrow function in these handlers
just void dispatch... should work

Reply:
Thanks, it works. I fixed it.